cinemy/cinemy/urls.py

```diff
@@ -25,4 +25,5 @@
     path("contact/", include("contact.urls")),
     path("user/", include("login.urls")),
     path("book_ticket/", include("cinema.urls")),
+    path("", include("cinema.urls")),
```

Comment:
It doesn't seem right to include the same URLs twice. Maybe book_ticket is not the proper term for this and you can use a different one more suited for all the URLs?

Reply:
I will try to reformat this piece of code.